### PR TITLE
[NeoOnnx] DepthToSpace operator

### PR DESCRIPTION
**Pull request by zimka**

### CODE REVIEW

File: NeoOnnx/src/Operators/DepthToSpaceOperator.h [hunk 1]
```diff
@@ -0,0 +1,35 @@
+/* Copyright © 2017-2020 ABBYY Production LLC
```
Comment (favorart) [inline NeoOnnx/src/Operators/DepthToSpaceOperator.h]: Please, change the first line of new created files from
`/* Copyright © 2017-2020 ABBYY Production LLC`
to
`/* Copyright © 2024 ABBYY`

File: NeoOnnx/src/Operators/DepthToSpaceOperator.cpp [hunk 2]
```diff
@@ -0,0 +1,74 @@
+/* Copyright © 2017-2020 ABBYY Production LLC
```
Comment (favorart) [inline NeoOnnx/src/Operators/DepthToSpaceOperator.cpp]: Please, change the first line of new created files from
`/* Copyright © 2017-2020 ABBYY Production LLC`
to
`/* Copyright © 2024 ABBYY`

File: NeoOnnx/src/Operators/DepthToSpaceOperator.cpp [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+#include "../common.h"
+#pragma hdrstop
+
+#include "NeoOnnxCheck.h"
+
+#include "onnx.pb.h"
+
+#include "DepthToSpaceOperator.h"
+
```
Comment (favorart) [inline NeoOnnx/src/Operators/DepthToSpaceOperator.cpp:24]: Please, add line here
`using namespace NeoML;`
before open namespace NeoOnnx.

File: NeoOnnx/src/Operators/DepthToSpaceOperator.h [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+// DepthToSpace operator
+class CDepthToSpaceOperator : public CLayerOperator {
+public:
+    CDepthToSpaceOperator( const onnx::NodeProto& depthToSpace, int opsetVersion );
+
+protected:
+    // CLayerOperator methods
+    void AddLayers( const CTensorArray& inputs, CDnn& dnn, CTensorArray& outputs ) const override;
+private:
```
Comment (favorart) [inline NeoOnnx/src/Operators/DepthToSpaceOperator.h:31]: Please, add an empty line before `private`

File: NeoOnnx/src/Operators/DropoutOperator.cpp [hunk 5]
```diff
@@ -80,3 +80,4 @@ float CDropoutOperator::getRatio( const CTensorArray& inputs ) const
 }
 
 } // namespace NeoOnnx
+
```
Comment (favorart) [inline NeoOnnx/src/Operators/DropoutOperator.cpp]: Delete it if no need

File: NeoOnnx/src/Operators/DepthToSpaceOperator.cpp (hunk 3 above)
Comment (zimka) [reply NeoOnnx/src/Operators/DepthToSpaceOperator.cpp:24]: Done

File: NeoOnnx/src/Operators/DepthToSpaceOperator.h (hunk 4 above)
Comment (zimka) [reply NeoOnnx/src/Operators/DepthToSpaceOperator.h:31]: Done

File: NeoOnnx/src/Operators/DepthToSpaceOperator.h (hunk 1 above)
Comment (zimka) [reply NeoOnnx/src/Operators/DepthToSpaceOperator.h]: Done

File: NeoOnnx/src/Operators/DropoutOperator.cpp (hunk 5 above)
Comment (zimka) [reply NeoOnnx/src/Operators/DropoutOperator.cpp]: Deleted

File: NeoOnnx/src/Operators/DepthToSpaceOperator.cpp (hunk 2 above)
Comment (zimka) [reply NeoOnnx/src/Operators/DepthToSpaceOperator.cpp]: Done

**Review by favorart — APPROVED** [review]

**Pull request merged**